packages/accounts-controller/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@metamask/accounts-controller",
+  "version": "1.0.0",
```

Comment:
we can set this to `0.0.0` and bump it in the first release to `1.0.0`

Reply:
Sure, we can set this to 0.0.0. Changed in 2c2fa52656e82694c8b4e48440e718cb8d3fd86d